sql/core/src/main/resources/org/apache/spark/sql/execution/ui/static/spark-sql-viz.js

```diff
@@ -82,7 +81,7 @@ function setupTooltipForSparkPlanNode(nodeId) {
  * and sizes of graph elements, e.g. padding, font style, shape.
  */
 function preprocessGraphLayout(g) {
-  g.graph().ranksep = "90";
+  g.graph().ranksep = "70";
```

Comment:
setting the value from 90 to 70 to make the graph neater.

Reply:
A gap between an arrow and a node in a whole-stage-codegen node seems to be broader than before (e.g. SerializeFromObject and its forwarding arrow in the screenshot).
It's a little bit ugly so I think it's better to narrow those gaps if you can.

Reply:
Got it.